src/utils/constants.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable no-undef */
+export const IS_TEST = process.env.NODE_ENV === "test";
```

Comment:
There's an existing constants file in /src/constants.js. It might be better put `IS_TEST` in there. There might be an argument for keeping test-specific constants in a separate file, but `IS_TEST` is one variable that's never going to be needed by test files.